packages/react-core/src/components/Form/__tests__/__snapshots__/FormGroup.test.tsx.snap

```diff
@@ -1,679 +1,35 @@
-          className="pf-c-form__label-text"
-        >
-          label
-        </span>
-      </label>
-       
-    </div>
-    <div
-      className="pf-c-form__group-control"
-    >
-      <input
-        id="id"
-      />
-      <div
-        aria-live="polite"
-        className="pf-c-form__helper-text pf-m-warning"
-        id="label-id-helper"
-      >
-        Validated FormGroup
-      </div>
-    </div>
-  </div>
-</FormGroup>
-`;
+exports[`FormGroup should render form group validated warning variant 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"id\\"><div class=\\"pf-c-form__helper-text pf-m-warning\\" id=\\"label-id-helper\\" aria-live=\\"polite\\">Validated FormGroup</div></div></div>"`;
 
-exports[`FormGroup component should render form group variant with function helperText 1`] = `
-<div>
-  <div
-    class="pf-c-form__group"
-  >
-    <div
-      class="pf-c-form__group-label"
-    >
-      <label
-        class="pf-c-form__label"
-        for="label-id"
-      >
-        <span
-          class="pf-c-form__label-text"
-        >
-          Label
-        </span>
-      </label>
-       
-    </div>
-    <div
-      class="pf-c-form__group-control"
-    >
-      <input
-        id="label-id"
-      />
-      <div>
-        label
-      </div>
-    </div>
-  </div>
-</div>
-`;
+exports[`FormGroup should render form group variant with function helperText 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"label-id\\"><span class=\\"pf-c-form__label-text\\">Label</span></label> </div><div class=\\"pf-c-form__group-control\\"><input id=\\"label-id\\"><div>label</div></div></div>"`;
 
-exports[`FormGroup component should render form group variant with function label 1`] = `
-<div>
-  <div
-    class="pf-c-form__group"
-  >
-    <div
-      class="pf-c-form__group-label"
-    >
-      <label
-        class="pf-c-form__label"
-        for="id"
-      >
-        <span
-          class="pf-c-form__label-text"
-        >
-          <div>
-            label
-          </div>
-        </span>
-      </label>
-       
-    </div>
-    <div
-      class="pf-c-form__group-control"
-    >
-      <input
-        aria-label="input"
-      />
-      
-    </div>
-  </div>
-</div>
-`;
+exports[`FormGroup should render form group variant with function label 1`] = `"<div data-testid=\\"form-group-test-id\\" class=\\"pf-c-form__group\\"><div class=\\"pf-c-form__group-label\\"><label class=\\"pf-c-form__label\\" for=\\"id\\"><span class=\\"pf-c-form__label-text\\"><div>label</div></span></label> </div><div class=\\"pf-c-form__group-control\\"><input aria-label=\\"input\\"></div></div>"`;
```

Comment:
again, not seeing the `.pf-c-form__helper-text`. Maybe i'm looking for the wrong thing?

Reply:
Same as the other test with the return function, it's just a div with the text `label`, which does exist in the snapshot.